daffodil-cli/src/it/scala/org/apache/daffodil/udf/TestCLIUdfs.scala

```diff
@@ -49,6 +50,8 @@ class TestCLIUdfs {
       }.toList.distinct
     commonPrefix
   }
+  lazy val savedParserFile = new File("savedParser.xsd.bin")
```

Comment:
Thoughts on creating this file via Files.createTempFile in the test? That way 1) it doesn't pollute the Daffodil directory so no one accidentally commits it and 2) it ensure there's a unique name so there aren't any issue where a subsequent test uses an old saved parser if somethign weird happens (e.g. save-parser fails, but continues the test with an old saved-parser).